__test__/api/comment-get.test.js

```diff
@@ -85,3 +85,29 @@ describeIfEndpoint(
     });
   }
 );
+
+describeIfEndpoint(
+  "GET",
+  "/v1/comments/:commentId/votes",
+  "GET '/v1/comments/:commentId/votes'",
```

Comment:
use appropriate string messages to denote the endpoint being tested `GET /comments/:commentId`